ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -268,7 +269,12 @@ objects:
           --quiet \
           --project "${GOOGLE_PROJECT_ID}" \
           --zone "${GOOGLE_COMPUTE_ZONE}" \
-          --recurse "${HOME}"/run-tests.sh packer@"${INSTANCE_PREFIX}":~/run-tests.sh.sh
+          --recurse "${HOME}"/run-tests.sh packer@"${INSTANCE_PREFIX}":~/run-tests.sh
+        LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute scp \
+          --quiet \
+          --project "${GOOGLE_PROJECT_ID}" \
+          --zone "${GOOGLE_COMPUTE_ZONE}" \
+          --recurse "${HOME}"/openshift-tests packer@"${INSTANCE_PREFIX}":~/openshift-tests
```

Comment:
`openshift-tests` is part of the container but not to the VM so to run it we need it to have it part of VM same as other bits.

Reply:
yup I hadn't gotten that far in awhile (installs were timing out and/or failing) to see this mistake, thanks!

Reply:
yup, it was being copied correctly, then I removed a while back when all tests were failing, when I added it back I did it too quickly, missed this, thanks! (couldn't ever actually run the tests to see this mistake until now!)